aea/cli/add.py

```diff
@@ -205,6 +201,23 @@ def _add_item_deps(
                 add_item(ctx, CONTRACT, contract_public_id)
 
 
+def fetch_item_remote(
+    ctx: Context, item_type: str, item_public_id: PublicId, dest_path: str
+) -> Path:
+    """Fetch item remote."""
+
+    if get_default_remote_registry() == REMOTE_IPFS:
+        try:
+            return fetch_ipfs(item_type, item_public_id, dest_path)
+        except HashNotProvided:
+            click.echo(f"Hash was not provided for: {item_public_id}")
+            click.echo("Will try with http repository.")
+
+    return fetch_package(
+        item_type, public_id=item_public_id, cwd=ctx.cwd, dest=dest_path,
+    )
```

Comment:
If fetching from IPFS fails should we try with http or let the user specify http while trying again?

Reply:
Sure let's try the other remote, why not